resources/migrations/001_update_migrations.yaml

```diff
@@ -6833,6 +6833,27 @@ databaseChangeLog:
               column:
                 name: user_id
 
+  - changeSet:
+      id: v51.2024-05-13T15:30:57
+      author: metamben
+      comment: Backup of dataset_query rewritten by the metrics v2 migration
+      changes:
+        - addColumn:
+            tableName: report_card
+            columns:
+              - column:
+                  name: dataset_query_metrics_v2_migration_backup
+                  remarks: The copy of dataset_query before the metrics v2 migration
+                  type: ${text.type}
```

Comment:
I was considering this too. Do you know of prior art?

Reply:
Yeah, my concern was serdes as well, just from the cleanliness of the serialized cards (in a git backed serdes system) I think it would probably be better separate if it's not too much more cumbersome

Reply:
I think creating a separate table is ultimately going to be more code and more noise, I think we should just have SerDes ignore the backup column entirely. It's only there in case we broke things anyway

Reply:
Populating the backup column with values can and should be done in Liquibase. Look at migration 85 for example. Only do stuff in Clojure migrations if they can't be done in SQL

Reply:
@camsaul, the custom migration populates this field only if it rewrites `dataset_query`. Are you suggesting always setting this field, or adding a condition approximating what's in the custom migration or something else?

Reply:
See my other comment, I think it's probably fine to just always set this Field since it's just copying pointers and we're already doing a full table scan anyway.

Reply:
Wouldn't that imply that on rollback we lose changes that we could keep?